README.md

```diff
@@ -57,6 +57,8 @@ The samples are built with Microsoft Compose libraries, [TwoPaneLayout](https://
 | | |
 | [NavigationRail](https://github.com/microsoft/surface-duo-compose-samples/tree/main/NavigationRail) | ![NavigationRail app icon](screenshots/navigation_rail.svg) | Nature-themed sample that shows how to use a combination of [NavigationRail](https://material.io/components/navigation-rail) and [BottomNavigation](https://material.io/components/bottom-navigation) when building apps for large screen and foldable devices in Jetpack Compose. |
 | | |
+| [DualView](https://github.com/microsoft/surface-duo-compose-samples/tree/main/DualView) | | Source editor sample that shows how to build the [Dual View](https://docs.microsoft.com/dual-screen/introduction#dual-view) app pattern in Jetpack Compose. |
```

Comment:
I think you need to update the name and link to SourceEditor (the app pattern is still correct though, it is the dual view app pattern)